packages/api/src/entities/Contribution.ts

```diff
@@ -10,6 +10,9 @@ export class Contribution extends Node<Contribution> {
   @Property({ columnType: 'text' })
   nodeID: string;
 
+  @Property({ columnType: 'text' })
+  authorGithubId: string;
```

Comment:
Let's change this to a reference to the `User` table instead; by doing that we'll have stronger associations and we can make a single call to get the fully populated `user` instead of needing to make separate calls to `user` and `contribution`